### PR TITLE
Jsolines

### LINKED ISSUE #863: Remove zero-area artifacts from exported vector isochrones

### PR DESCRIPTION
**Pull request by trevorgerhardt**
Updated Jsolines and all other packages that had updates (using npm-check-updates). In turn, updated snapshots, eslint, and re-formatted with new prettier defaults.

While there are a lot of changes the only functional change should be the removal of 2D holes from the Jsolines update.

closes #863

### CODE REVIEW

File: lib/actions/analysis/index.js [hunk 1]
```diff
@@ -209,24 +209,20 @@ export const handleSurface = (error, responses) => {
...
-      setScenarioApplicationWarnings([
-        ...surface.warnings,
-        ...(comparisonSurface ? comparisonSurface.warnings : [])
-      ]),
-      setTravelTimeSurface(surface),
-      setComparisonTravelTimeSurface(comparisonSurface),
-      setIsochroneFetchStatus(false)
-    ]
-  } catch (e) {
-    throw e
```
Comment (ansoncfit) [inline lib/actions/analysis/index.js]: How is this error handled now?

File: lib/components/link.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {routeTo} from 'lib/router'
 export default function Link({children, to, ...p}) {
   const {as, href, query} = routeTo(to, p)
   return (
-    <NextLink as={as} href={{pathname: href, query}} passHref prefetch>
+    <NextLink as={as} href={{pathname: href, query}} passHref>
```
Comment (ansoncfit) [inline lib/components/link.js]: What's the effect of this change?

File: lib/modules/admin/selectors.js [hunk 3]
```diff
@@ -6,37 +6,29 @@ export const jobs = state => state.admin.jobs
...
-
-export const allBundlesById = createSelector(
-  workers,
-  workers => {
-    const bundles = {}
-    workers.forEach(w =>
-      (w.bundles || []).forEach(b => {
-        bundles[b._id] = b
-      })
+export const jobsWithWorkers = createSelector(jobs, workers, (jobs, workers) =>
```
Comment (ansoncfit) [inline lib/modules/admin/selectors.js]: Interesting coincidence that our default accessibility analysis is connecting workers to jobs.

File: lib/utils/component.js [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
 import {createRouter} from 'next/router'
-import {RouterContext} from 'next-server/dist/lib/router-context'
+import {RouterContext} from 'next/dist/next-server/lib/router-context'
 import React from 'react'
 import {Map} from 'react-leaflet'
 import {Provider} from 'react-redux'
 
 import {makeMockStore, mockStores} from './mock-data'
 
+global.Math.random = () => 0.5
```
Comment (ansoncfit) [inline lib/utils/component.js]: What is this used for?

File: lib/components/link.js (hunk 2 above)
Comment (trevorgerhardt) [reply lib/components/link.js]: The updated version of `next.js` does not require this anymore. So there should be no effect.

File: lib/utils/component.js (hunk 4 above)
Comment (trevorgerhardt) [reply lib/utils/component.js]: Next.js creates IDs that are used for page versions and show up in snapshots. I meant to add a comment once that actually worked. Will do.

File: lib/actions/analysis/index.js (hunk 1 above)
Comment (trevorgerhardt) [reply lib/actions/analysis/index.js]: The try/catch was leftover from old code that used the error. Catching and rethrowing immediately is the same as not catching at all. The new version of ESLint picked up on that and showed a warning.

**Pull request merged**